dts/bindings/clock/st,stm32f105-pll-clock.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2021, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Main PLL node binding for Connectivity line devices (STM32F105/STM32F107)
+
+  Takes one of clk_hse, pll2 or clk_hsi as input clock.
+  When clk_hsi is used a fixed prescaler is applied. When input clock is hse or
+  pll2, configurable prescaler is used.
+
+  Up to 2 output clocks could be supported and for each output clock, the
+  frequency can be computed with the following formula:
+
+    f(PLLCLK) = f(PLLIN) x PLLMUL   --> SYSCLK (System Clock)
+    f(USBCLK) = f(PLLCLK) / USBPRE  --> USB
+
+    with, depending on the case:
+            f(PLLIN) = f(input_clk) / 2       if input_clk = clk_hsi
+            f(PLLIN) = f(input_clk) / PREDIV  if input_clk = clk_hse or pll2
+
+  The PLL output frequency must not exceed 72 MHz.
+
+
+compatible: "st,stm32f105-pll-clock"
+
+include: [clock-controller.yaml, base.yaml]
+
+properties:
+    "#clock-cells":
+      const: 0
+
+    clocks:
+      required: true
+
+    mul:
+      type: int
+      required: true
+      description: |
+          Main PLL multiplication factor for VCO
+      enum:
+        - 4   # x4
+        - 5   # x5
+        - 6   # x6
+        - 7   # x7
+        - 8   # x8
+        - 9   # x9
+        - 13  # x6.5
```

Comment:
is `13` chosen for any reason? maybe a comment in description would clarify it

Reply:
13 is the register value to be applied. I'm adding a clarification in description field

Reply:
If it is register value then other value are wrong: x4 should be register value 2
```
Bits 21:18 PLLMUL[3:0]: PLL multiplication factor
     These bits are written by software to define the PLL multiplication factor. They can be written only
     when PLL is disabled.
     000x: Reserved
     0010: PLL input clock x 4
     0011: PLL input clock x 5
     0100: PLL input clock x 6
     0101: PLL input clock x 7
     0110: PLL input clock x 8
     0111: PLL input clock x 9
     10xx: Reserved
     1100: Reserved
     1101: PLL input clock x 6.5
     111x: Reserved
```

Reply:
Remark still valid